src/widgets/barchart/bar_group.rs

```diff
@@ -35,6 +39,23 @@ impl<'a> BarGroup<'a> {
     pub(super) fn max(&self) -> Option<u64> {
         self.bars.iter().max_by_key(|v| v.value).map(|v| v.value)
     }
+
+    pub(super) fn render_label(self, buf: &mut Buffer, area: Rect, default_label_style: Style) {
+        if let Some(mut label) = self.label {
+            // patch label styles
+            for span in &mut label.spans {
+                span.style = default_label_style.patch(span.style);
+            }
+
+            let x_offset = match label.alignment {
+                Some(Alignment::Center) => area.width.saturating_sub(label.width() as u16) >> 1,
+                Some(Alignment::Right) => area.width.saturating_sub(label.width() as u16),
+                _ => 0,
+            };
```

Comment:
just to communicate clearly. This is a breaking change. It will allow the user to decide, where to render labels of the group. As a result the labels will be rendered Left by default.

I can move this to another PR if needed

Reply:
(this is fine as part of this PR). Resolving.